addons/web/static/src/js/views/list/list_editable_renderer.js

```diff
@@ -359,7 +361,7 @@ ListRenderer.include({
      * @returns {boolean}
      */
     isEditable: function () {
-        return this.editable || this.selection.length;
+        return this.editable || (this.selection.length && this.isMultiEditable) ;
```

Comment:
The `_isRecordEditable` function also needs to be edited with this option since it works independently from `isEditable`